projects/angular/src/data/datagrid/datagrid-footer.ts

```diff
@@ -19,6 +20,7 @@ import { DetailService } from './providers/detail.service';
         <clr-checkbox-wrapper class="datagrid-footer-select">
           <input clrCheckbox type="checkbox" checked="checked" disabled />
           <label>{{ selection.current.length }}</label>
+          <span class="clr-sr-only">{{ commonStrings.keys.selectedRows }}</span>
```

Comment:
This addresses the use case where a screen reader needs to output meaningful content when this is focused on.